docs/source/installing_docker.rst

```diff
@@ -128,14 +128,14 @@ To upgrade running Kiwi TCMS containers execute the following commands::
     # !!! docker tag to keep older image version on the machine
     docker-compose pull # to fetch latest versions from Docker Hub
     docker-compose up -d
-    docker exec -it kiwi_web /Kiwi/manage.py migrate
+    docker exec -it kiwi_web /Kiwi/manage.py upgrade
```

Comment:
Replace `manage.py migrate` with `manage.py upgrade` inside `tests/migrations-rollback` for a limited test coverage of this command.